stress_test/controller.py

```diff
@@ -41,11 +38,16 @@ def __init__(self, ctrl_base_dir, test_config):
         self.of_port = test_config['controller_port']
         self.logs_dir = self.base_dir + test_config['controller_logs_dir']
 
-        self.build_hnd = self.base_dir + test_config['controller_build_handler']
-        self.start_hnd = self.base_dir + test_config['controller_start_handler']
-        self.stop_hnd = self.base_dir + test_config['controller_stop_handler']
-        self.status_hnd = self.base_dir + test_config['controller_status_handler']
-        self.clean_hnd = self.base_dir + test_config['controller_clean_handler']
+        self.build_hnd = self.base_dir + test_config['controller_'
```

Comment:
Do not split a single string, it worsens a lot code readability. 

There are two choices here:
1. either leave the whole statement as is, without further splitting it
2. either split the line at some operator, e.g. `+`

I would prefer option 1, even if it violates the line-length limits

Check this everywhere